src/launchoptions.rs

```diff
@@ -81,7 +81,7 @@ impl LaunchOptionsV2 {
             }
         }
 
-        let launch_options_regex = Regex::new(r#"\t{6}"LaunchOptions"\t{2}"([(\-\w)\s]*)""#)
+        let launch_options_regex = Regex::new(r#"\t{6}"LaunchOptions"\t{2}"([(\-\w\%\!\@\^\&)\s]*)""#)
```

Comment:
So i agree that not all of those need to be escaped, but escaping a nonspecial character should just yield that character so i was being safe over sorry. But also no, the group is inside a `[...]` block, so this is per character, rather than per word

Reply:
Ah alright, I was mistakenly assuming that putting it inside a capture group would take it as a whole word instead, but I think it's actually just matching '(' and ')' by being inside `[...]`. Actually why are there parentheses in there?